fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -3731,6 +3731,10 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws UserExcep
         if (replicaAlloc.isNotSet()) {
             replicaAlloc = ReplicaAllocation.DEFAULT_ALLOCATION;
         }
+        if (replicaAlloc.getTotalReplicaNum() < Config.min_table_replication_num) {
```

Comment:
This should be done in `PropertyAnalyzer.analyzeReplicaAllocation()`.
Because there are many other place to set or modify the replication number.

Reply:
done